CHANGELOG.md

```diff
@@ -21,6 +21,9 @@ This is a changelog for Piwik platform developers. All changes for our HTTP API'
 ### Internal Change
 * The option `branch` of the console command `development:sync-system-test-processed` was removed as it is no longer needed.
 
+### APIs
+* There is a new event `CronArchive.getIdSitesNotUsingTracker` that allows you to set a list of idSites that do not use the Tracker API to make sure we archive these sites if needed.
```

Comment:
It's not that important, but I think `CronArchive.getIdSitesToForceArchivingFor` would be a better name (since it is explicit about what the site IDs will be used for). `getIdSitesNotUsingTracker` I think requires knowing how CronArchive works.

Also, I don't think this event is enough to force archiving for the sites, since if there are no visits archiving will be skipped for them (see: https://github.com/piwik/piwik/blob/master/core/ArchiveProcessor/Loader.php#L121). This is a problem for plugins that display 3rd party data, for example. Should be noted here and the event docs.

OR we could call this event in the core archiver to force archiving even if there are no visits for that site... though perhaps this clashes w/ MetaSites. In this case, a new event could be created. Anyway, this hypothetical new event isn't required for this PR.